operations/template/alert.tf

```diff
@@ -101,9 +101,10 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "rs_sftp_log_errors_alert
 
   query = <<-QUERY
       AppServiceConsoleLogs
+      | where _ResourceId !contains "pre-live"
       | project columnifexists("ResultDescription", 'default_value')
       | project  JsonResult = parse_json(ResultDescription)
-      | evaluate bag_unpack(JsonResult) : (level: string, msg: string)
+      | evaluate bag_unpack(JsonResult) : (level: string, msg: string, ["@timestamp"]: string)
```

Comment:
It isn't `@timestamp` in SFTP Ingestion.  At least, I don't think it is.  It is for TI, though.